presidio-anonymizer/presidio_anonymizer/core/text_replace_builder.py

```diff
@@ -31,12 +31,12 @@ def get_text_in_position(self, start: int, end: int) -> str:
         return self.output_text[start:end]
 
     def replace_text_get_insertion_index(
-        self, anonymized_text: str, start: int, end: int
+            self, new_text: str, start: int, end: int
```

Comment:
```suggestion
            self, replacement_text: str, start: int, end: int
```
I believe this name is more meaningful. wdyt?

Reply:
Done.